utils/cliutils/commandsflags.go

```diff
@@ -1049,6 +1060,18 @@ var flagsMap = map[string]cli.Flag{
 		Name:  audience,
 		Usage: "[Optional] A space-separate list of the other Artifactory instances or services that should accept this token identified by their Artifactory Service IDs, as obtained by the 'jfrog rt curl api/system/service_id' command.` `",
 	},
+	csv: cli.StringFlag{
+		Name:  csv,
+		Usage: "[Mandatory] Path to a csv file contains users details. The first row of the file should contain the user fields names e.g name,password,email` `",
```

Comment:
[Mandatory] Path to a csv file contains users details. The first row of the file should contain the user fields names e.g name,password,email
-->
[Mandatory] Path to a csv file with the users' details. The first row of the file should contain the user fields names e.g name,password,email